app/models/plugins/fundraiser.rb

```diff
@@ -7,7 +7,23 @@ class Plugins::Fundraiser < ActiveRecord::Base
   DEFAULTS = { title: 'Donate now' }
 
   def liquid_data(supplemental_data={})
-    bands = donation_band.present? ? donation_band.internationalize.to_json : "null"
+    donation_band_name = ''
+    backup_id = nil
+
+    if supplemental_data.has_key? :url_params
+      donation_band_name = supplemental_data[:url_params][:donation_band]
+    end
+    if donation_band.present?
+      backup_id = donation_band.id
```

Comment:
Rails has `#try` for situations like this: `backup_id = donation_band.try(:id)` - it'll return nil if `donation_band` is undefined.